azure-bundle/LICENSE

```diff
@@ -201,319 +201,333 @@
-License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+Project URL (from POM): https://github.com/java-native-access/jna
+License (from POM): Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+License (from POM): LGPL-2.1-or-later - https://www.gnu.org/licenses/old-licenses/lgpl-2.1
 
 --------------------------------------------------------------------------------
 
-Group: net.minidev  Name: accessors-smart  Version: 2.4.9
-Project URL: https://urielch.github.io/
-License: The Apache Software License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
+Group: net.minidev  Name: accessors-smart  Version: 2.5.1
+Project URL (from manifest): https://urielch.github.io/
+Project URL (from POM): https://urielch.github.io/
+License (from POM): The Apache Software License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: net.minidev  Name: json-smart  Version: 2.4.10
-Project URL: https://urielch.github.io/
-License: The Apache Software License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
+Group: net.minidev  Name: json-smart  Version: 2.5.1
+Project URL (from manifest): https://urielch.github.io/
+Project URL (from POM): https://urielch.github.io/
+License (from POM): The Apache Software License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: org.codehaus.woodstox  Name: stax2-api  Version: 4.2.1
-Project URL: http://github.com/FasterXML/stax2-api
-License: The BSD License - http://www.opensource.org/licenses/bsd-license.php
+Group: org.ow2.asm  Name: asm  Version: 9.6
+Project URL (from manifest): http://asm.ow2.org
+Manifest License: BSD-3-Clause;link=https://asm.ow2.io/LICENSE.txt (Not packaged)
+Project URL (from POM): http://asm.ow2.io/
+License (from POM): BSD-3-Clause - https://asm.ow2.io/license.html
+License (from POM): The Apache Software License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-Group: org.ow2.asm  Name: asm  Version: 9.3
-Project URL: http://asm.ow2.io/
-License: BSD-3-Clause - https://asm.ow2.io/license.html
-License: The Apache Software License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
+Group: org.reactivestreams  Name: reactive-streams  Version: 1.0.4
+Project URL (from manifest): http://reactive-streams.org
+Project URL (from POM): http://www.reactive-streams.org/
+License (from POM): MIT-0 - https://spdx.org/licenses/MIT-0.html
 
 --------------------------------------------------------------------------------
 
-Group: org.reactivestreams  Name: reactive-streams  Version: 1.0.4
-Project URL: http://www.reactive-streams.org/
-License: MIT-0 - https://spdx.org/licenses/MIT-0.html
+Group: org.slf4j  Name: slf4j-api  Version: 2.0.16
```

Comment:
Imho, that's not correct here: slf4j-api is excluded (https://github.com/apache/iceberg/blob/main/azure-bundle/build.gradle#L43), and not in the distributed jar file, so it should not be in `LICENSE`.

See this PR: https://github.com/apache/iceberg/pull/12052